pkg/build/strategies/sti/postexecutorstep.go

```diff
@@ -496,6 +502,15 @@ func downloadAndExtractFileFromContainer(docker dockerpkg.Docker, tar s2itar.Tar
 	return utilstatus.NewFailureReason("", ""), nil
 }
 
+func checkLabelSize(labels map[string]string) error {
+	for k, v := range labels {
+		if len(v) > maximumLabelSize {
```

Comment:
i think this is supposed to be checking the total size of all the labels, not the size of each individual label.